effects/data_base_effect.py

```diff
@@ -0,0 +1,71 @@
+from typing import List
+
+import message_text_config as config
+import user_interaction
+import utils
+from effects.effect import Effect
+from message_text_config import Errors
+
+
+class DataBase(Effect):
+    def __init__(self, game: Game, name: str, creator: Citizen) -> None:
+        super(DataBase, self).__init__(game, name, creator, 4)
```

Comment:
You can use only 'super().__init__' instead of super(DataBase, self).__init__
No need for writing classname explicitly